source/fundamentals/collations.txt

```diff
@@ -4,5 +4,295 @@ Collations
+
+To use the index in an operation, your operation must use the same collation as the one
+specified in the index. Additionally, ensure that the operation is covered by the index that
+contains the collation. Set a collation through an ``IndexOptions`` object and call the
+``CreateOne()`` method with your options object as an argument.
+
+Example
+~~~~~~~
+
+After creating the ``books`` collection and applying a default collation, as shown in the
+:ref:`golang-create-collection` section, you cannot change the collection's default collation.
+However, you can create an index for the collection with a different collation. 
+
+The following example uses the ``CreateOne()`` method to create an ascending index on the
+``name`` field and specifies a new collation with an ``"en_US"`` locale:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+       myCollation := &options.Collation{Locale: "en_US"}
+       opts := options.Index().SetCollation(myCollation)
+
+       indexModel := mongo.IndexModel{
+         Keys:    bson.D{{"name", 1}},
+         Options: opts,
+       }
+
+       name, err := coll.Indexes().CreateOne(context.TODO(), indexModel)
+       if err != nil {
+         panic(err)
+       }
+       fmt.Println("Name of Index Created: " + name)
+
+   .. output::
+      :language: none
+      :visible: false
+
+       Name of Index Created: name_1
+
+.. _golang-op-collation:
+
+Set a Collation on an Operation
+-------------------------------
+
+Operations that read, update, and delete documents from a collection can use collations.
+Applying a collation to an operation overrides any default collation previously defined
+for a collection. 
+
+If you apply a new collation to an operation that differs from an index's collation,
+you cannot use that index. As a result, the operation may not perform as well as one
+that is covered by an index. For more information on the disadvantages of sorting operations
+not covered by an index, see :manual:`Using Indexes to Sort Query Results </tutorial/sort-results-with-indexes/>`.
+See the :manual:`MongoDB manual </reference/collation/#collation-document>` for a list of
+operations that support collation.
+
+Example
+~~~~~~~
+
+You can use operations that support collation to update and query documents in the 
+``books`` collection. 
+
+The following example uses the ``Find()`` method to return documents with ``length`` 
+values greater than ``"1000"``. The ``NumericOrdering`` collation field has a value of
+``true`` to ensure that values are sorted in numerical order rather than alphabetical
+order:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+      filter := bson.D{{"length", bson.D{{"$gt", "1000"}}}}
+      myCollation := &options.Collation{Locale: "en_US", NumericOrdering: true}
+      opts := options.Find().SetCollation(myCollation)
+
+      cursor, err := coll.Find(context.TODO(), filter, opts)
```

Comment:
Missing an error check below this `Find`.

Reply:
Added this